### PR TITLE
Implement auto-assigned ticketing

### PR DESCRIPTION
**Pull request by johndmulhausen**
This PR changes the "file a ticket" link so that it automatically assigns the user's ticket to the writer who is covering the docs in that directory. 

CC @sanscontext @joaofnfernandes @mstanleyjones @londoncalling

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview processing.

Built with commit edf553f89235eb2a677a2c53fed018eeb3a56489

https://app.netlify.com/sites/docker-docs/deploys/582501ba66867431d054da45

**Comment by mstanleyjones** [comment]
LGTM, things under `engine` seem to be assigned to me.

Note that this might break as we reorg to be less org-chart organized.

File: _config.yml [hunk 1]
```diff
@@ -27,3 +27,68 @@ defaults:
...
+  - scope:
+      path: "cs-engine"
+    values:
+      assignee: "joaofnfernandes"
+  - scope:
+      path: "datacenter"
+    values:
+      assignee: "joaofnfernandes"
+  - scope:
+      path: "cs-engine"
```
Comment (joaofnfernandes) [inline _config.yml]: Looks like there's a duplicate entry for `cs-engine`.

**Comment by joaofnfernandes** [comment]
LGTM

File: _config.yml [hunk 2]
```diff
@@ -27,3 +27,68 @@ defaults:
...
+    values:
+      assignee: "londoncalling"
+  - scope:
+      path: "docker-for-windows"
+    values:
+      assignee: "londoncalling"
+  - scope:
+      path: "docker-hub"
+    values:
+      assignee: "johndmulhausen"
```
Comment (sanscontext) [inline _config.yml]: This should be me as well I've been maintaining those docs.

**Comment by johndmulhausen** [comment]
@joaofnfernandes @sanscontext Addressed feedback, WDYT?

**Comment by johndmulhausen** [comment]
Feedback handled, thanks guys!

**Pull request merged**